constants/src/lib.rs

```diff
@@ -1,7 +1,11 @@
 pub const VENDOR_IDENTIFIER: &str = "MongoDB";
 
 // SQL states
-pub const HYC00: &str = "HYC00";
-pub const HY000: &str = "HY000";
-pub const HY024: &str = "HY024";
-pub const _01S02: &str = "01S02";
+pub const NOT_IMPLEMENTED: &str = "HYC00";
+pub const TIMEOUT_EXPIRED: &str = "HYT00";
+pub const GENERAL_ERROR: &str = "HY000";
+pub const INVALID_VALUE: &str = "HY024";
```

Comment:
[nit] What about naming it INVALID_ATTR_VALUE instead of INVALID_VALUE so that it is clear that this is related to a value from a result set?

Reply:
Sounds good, made the naming change to `INVALID_ATTR_VALUE`